### PR TITLE
Fix names of certain files, resolve lack of documentation of mod options

### PR DESCRIPTION
**Pull request by AmenneHolelane**
Large PR for readme update and file changes

### CODE REVIEW

**Comment by AmenneHolelane** [comment]
hi sorry, I got bored at work LOL 

add preview options with appropriate names for pictures.

change Folder name from partysheild-deep-blue to partyshield-deep-blue 

change file name from discord-selected-window-border.png to black-selected-window-border.png
change file name from discord-non-selected-window-transparency.png to black-non-selected-window-transparency.png

After the PR is complete, i'll change the links to reference your github but i need the file changes merged before i can change the links lol

File: README.md [hunk 1]
```diff
@@ -49,7 +49,109 @@ There are also a lot of other things that have been changed. Check out the in-ga
...
+#### Deep blue shield bar
+![Deep blue shield bar](https://github.com/AmenneHolelane/ffxiv-material-ui/blob/master/ModPacks/OPTIONS/02_PartyList/05_partyshield-deep-blue/partyshield-deep-blue.png)
+
+### Icons
+
+#### Rounded corner job icons
+![Round background job icons](https://github.com/AmenneHolelane/ffxiv-material-ui/blob/master/ModPacks/OPTIONS/03_Icon/01_round-background-job-icon/round_background_job_icon.png)
+
+#### Squared corner job icons
+![edge background job icons](https://github.com/AmenneHolelane/ffxiv-material-ui/blob/master/ModPacks/OPTIONS/03_Icon/02_edge-background-job-icon/02-1_edge_background_job_icon.png)
```
Comment (AmenneHolelane) [inline README.md]: the english translations of these options were a little strange.

File: README.md [hunk 2]
```diff
@@ -49,7 +49,109 @@ There are also a lot of other things that have been changed. Check out the in-ga
...
+#### Black style nonselected window transparency
+![nonselected window transparency](https://github.com/AmenneHolelane/ffxiv-material-ui/blob/master/ModPacks/OPTIONS/05_Window/04_black-non-selected-window-transparency/black-non-selected-window-transparency.png)
+
+### Works with both
+
+#### Discord style chatlog resize symbol
+![Discord chatlog resize symbol](https://github.com/AmenneHolelane/ffxiv-material-ui/blob/master/ModPacks/OPTIONS/05_Window/05_discord-chatlog-resize-symbol/discord-chatlog-resize-symbol.png)
+
+#### Black style chatlog resize symbol
+![Black chatlog resize symbol](https://github.com/AmenneHolelane/ffxiv-material-ui/blob/master/ModPacks/OPTIONS/05_Window/06_black-chatlog-resize-symbol/black-chatlog-resize-symbol.png)
```
Comment (AmenneHolelane) [inline README.md]: could you confirm that these specifically are correct? There isn't nothing documenting what works with Black only or Discord only so I went by what had the filename specifying and assumed that if there wasn't a file it worked with both

**Comment by AmenneHolelane** [comment]
should resolve #144

**Comment by AmenneHolelane** [comment]
add clarification for which options are style specific and which are not

**Comment by skotlex** [comment]
I rewrote the document in a new form. The data was used for reference purposes, but I will not approve it.

**Pull request closed without merging**